yarn.lock

```diff
@@ -1330,11 +1330,16 @@
     "@babel/helper-validator-identifier" "^7.22.5"
     to-fast-properties "^2.0.0"
 
-"@braintree/sanitize-url@=6.0.2", "@braintree/sanitize-url@^6.0.0":
+"@braintree/sanitize-url@=6.0.2":
```

Comment:
why change the lock if there are no changes in the package.json?

Reply:
Reverted: https://github.com/zeta-chain/docs/pull/195/commits/db0f01f6a8fe48efd8a3006e98e0ef364f8da37b